src/chainparams.cpp

```diff
@@ -486,7 +486,7 @@ class CTestNetParams : public CChainParams {
 
         // Deployment of Deployment of Basic BLS, AssetLocks, EHF
         consensus.vDeployments[Consensus::DEPLOYMENT_V19].bit = 8;
-        consensus.vDeployments[Consensus::DEPLOYMENT_V19].nStartTime = 19999999999;     // TODO: To be determined later
+        consensus.vDeployments[Consensus::DEPLOYMENT_V19].nStartTime = 1677024000;     // Wed, Feb 22, 2023 0:00:00
```

Comment:
I like this number better :stuck_out_tongue: 
```suggestion
        consensus.vDeployments[Consensus::DEPLOYMENT_V19].nStartTime = 1680220800;     // Fri, Mar 31, 2023 0:00:00
```

Reply:
Confirmed accurate